ui/raidboss/emulator/data/network_log_converter/LineEvent.ts

```diff
@@ -1,45 +1,50 @@
 import EmulatorCommon from '../../EmulatorCommon';
+import LogRepository from './LogRepository';
 
 /**
  * Generic class to track an FFXIV log line
  */
 export default class LineEvent {
-  constructor(repo, line, parts) {
-    this.offset = 0; // To be calculated later
-
-    this.parts = parts;
-
-    this.decEvent = parseInt(this.parts[0]);
+  public offset = 0;
+  decEvent: number;
+  hexEvent: string;
+  timestamp: number;
+  networkLine: string;
+  convertedLine = '';
+  invalid = false;
+  index = 0;
+  constructor(repo: LogRepository, line: string, public parts: string[]) {
```

Comment:
```suggestion

  constructor(repo: LogRepository, line: string, public parts: string[]) {
```

I can't find an eslint rule for this, but I'd appreciate a linebreak between members and functions.

Reply:
Added line break for next commit. As for an eslint rule, maybe this one could be configured? I'm not very well versed in eslint configuration.

https://eslint.org/docs/rules/padding-line-between-statements

Reply:
Thanks, I'll take a look.